### ISSUE TITLE
PE regression?

### ISSUE THREAD

**Issue by CannedGeo** [seg/2016-ml-contest#15]
Hi all- Has anyone tried generating a PE for the wells that do not have it (Alexander D and Kimzey A)? 
I have made some attempts using `from sklearn.svm import SVR` and playing around with the different models. Do you think this would lead to a valid answer? Are there better regression techniques to use?

I started a repo for PE regression that includes a [notebook](https://github.com/CannedGeo/SEG-ML-exercise-PE-regression/blob/master/PE_regression_adapted_from%20SEG_TLE.ipynb) I have been playing with. To call it sloppy is probably an understatement, but its there as a work in progress.

**Comment by CannedGeo** [comment]
If I can come up with a solution that's worth further discussion, maybe this can be looked at closer. Will close.

**Comment by ShiangYong** [comment]
Hi CannedGeo, in my submission I used a convolutional neural network to do PE regression. Comparing my classifier using imputed PE via regression against just using mean value of PE, I obtained higher F1 scores with the regression PE.

ar4's submission also confirmed this.

**Comment by CannedGeo** [comment]
Hi @ShiangYong, thanks for the input. It definitely seems more helpful to use a regression model for the PE instead of mean.  I also had decent LOGO results using gradient boost regressor. Good luck!
